server/src/main/java/org/apache/druid/server/coordinator/BalancerSegmentHolder.java

```diff
@@ -19,29 +19,28 @@
 
 package org.apache.druid.server.coordinator;
 
-import org.apache.druid.client.ImmutableDruidServer;
 import org.apache.druid.timeline.DataSegment;
 
 /**
  */
 public class BalancerSegmentHolder
```

Comment:
Change summary:
Added `ServerHolder` to avoid having to find the corresponding peon later in the flow.